libanki/src/main/java/com/ichi2/anki/libanki/Tags.kt

```
@@ -80,6 +80,28 @@ class Tags(
         )
 
+    /** Update tags for the given notes. Notes not in the collection are ignored. */
+    @LibAnkiAlias("bulk_update")
+    fun bulkUpdate(
+        noteIds: List<NoteId>,
+        tags: String,
+    ): OpChanges {
+        val notes = noteIds.mapNotNull {
+            try {
+                col.getNote(it)
+            } catch (e: RuntimeException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)